.gitattributes

```diff
@@ -38,68 +38,81 @@
 
-/experimental            ocaml-typo=prune
+emacs/*.el               typo.long-line=may
+emacs/caml.el            typo.long-line=may typo.missing-header
+emacs/COPYING            typo.tab typo.non-printing typo.missing-header
+emacs/ocamltags.in       typo.non-printing
 
-/manual                  ocaml-typo=prune
+/experimental            typo.prune
 
-ocamldoc/Changes.txt     ocaml-typo=missing-header
-ocamldoc/ocamldoc.sty    ocaml-typo=missing-header
-ocamldoc/odoc_index.html ocaml-typo=missing-header
+/manual                  typo.prune
 
-otherlibs/win32unix/readlink.c    ocaml-typo=long-line
-otherlibs/win32unix/stat.c        ocaml-typo=long-line
-otherlibs/win32unix/symlink.c     ocaml-typo=long-line
+ocamldoc/**              typo.long-line=may
+ocamldoc/Changes.txt     typo.missing-header
+ocamldoc/ocamldoc.sty    typo.missing-header
+ocamldoc/odoc_index.html typo.missing-header
 
-runtime/i386.S            ocaml-typo=long-line
+otherlibs/win32unix/readlink.c    typo.long-line
+otherlibs/win32unix/stat.c        typo.long-line
+otherlibs/win32unix/symlink.c     typo.long-line
 
-stdlib/hashbang     ocaml-typo=white-at-eol,missing-lf
+runtime/i386.S      typo.long-line
 
-testsuite/tests/**                                      ocaml-typo=missing-header
-testsuite/tests/lib-bigarray-2/bigarrf.f                ocaml-typo=missing-header,tab
-testsuite/tests/lib-unix/win-stat/fakeclock.c           ocaml-typo=
-testsuite/tests/misc-unsafe/almabench.ml                ocaml-typo=missing-header,long-line
-testsuite/tests/tool-toplevel/strings.ml                ocaml-typo=utf8,missing-header
-testsuite/tests/typing-unboxed-types/test.ml.reference* ocaml-typo=white-at-eof,missing-header
-testsuite/tests/win-unicode/*.ml                        ocaml-typo=utf8,missing-header
-testsuite/tools/*.S                                     ocaml-typo=missing-header
-testsuite/tools/*.asm                                   ocaml-typo=missing-header
-testsuite/typing                                        ocaml-typo=missing-header
+stdlib/hashbang     typo.white-at-eol typo.missing-lf
+
+testsuite/tests/**                                      typo.missing-header
+testsuite/tests/lib-bigarray-2/bigarrf.f                typo.tab
+testsuite/tests/lib-unix/win-stat/fakeclock.c           typo.missing-header=false
+testsuite/tests/misc-unsafe/almabench.ml                typo.long-line
+testsuite/tests/tool-toplevel/strings.ml                typo.utf8
+testsuite/tests/typing-unboxed-types/test.ml.reference* typo.white-at-eof
+testsuite/tests/win-unicode/*.ml                        typo.utf8
+testsuite/tools/*.S                                     typo.missing-header
+testsuite/tools/*.asm                                   typo.missing-header
+testsuite/typing                                        typo.missing-header
+
+# prune testsuite reference files
+testsuite/tests/**/*.reference               typo.prune
+testsuite/tests/**/reference                 typo.prune
```

Comment:
This is used by *just* the `lib-scanf-2` test - can't we just rename the file and get rid of the special case?

Reply:
Seconded. That should slightly simplify the test script itself.

Reply:
Done.